src/verifier/snp/mod.rs

```diff
@@ -0,0 +1,141 @@
+use anyhow::{anyhow, Context, Result};
+extern crate serde;
+use self::serde::{Deserialize, Serialize};
+use super::*;
+use async_trait::async_trait;
+use serde_json::json;
+use sev::firmware::guest::types::AttestationReport;
+use sev::firmware::host::types::{CertTable, SnpCertType};
+
+use openssl::{
+    ec::EcKey,
+    ecdsa,
+    pkey::{PKey, Public},
+    x509,
+};
+use sha2::{Digest, Sha384};
+
+#[derive(Serialize, Deserialize)]
+struct SnpEvidence {
+    attestation_report: AttestationReport,
+    cert_chain: CertTable,
+}
+
+#[derive(Debug, Default)]
+pub struct Snp {}
```

Comment:
I am not seeing this used anywhere. Is this just a place-holder for the moment, or just to tie the impl to a generic?